src/app/components/ShortAddress/__tests__/index.test.tsx

```diff
@@ -1,13 +1,35 @@
 import { render, screen } from '@testing-library/react'
 
-import { ShortAddress } from '..'
+import { trimLongString, ShortAddress } from '..'
+
+describe('trimLongString', () => {
+  it('should return trimmed string', () => {
+    expect(trimLongString('qwertyuiopasdfghjkl')).toEqual('qwertyuiop...sdfghjkl')
+    expect(trimLongString('qwertyuiopasdfghjkl', 2, -2)).toEqual('qw...kl')
+  })
+
+  it('should return short hash', () => {
+    expect(trimLongString('6557c86267761b845aa00dac39e892ca58922305c883965f4b0b67a009b0d8d3')).toEqual(
+      '6557c86267...09b0d8d3',
+    )
+  })
+
+  it('should return original string if it is shorter than trimStart value', () => {
+    expect(trimLongString('foo')).toEqual('foo')
```

Comment:
This isn't generalized for short strings yet:
`trimLongString('Unavailable')` returns `'Unavailabl...vailable'`

Reply:
https://github.com/oasisprotocol/oasis-wallet-web/issues/950